src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SemanticFacts/CSharpSemanticFacts.cs

```diff
@@ -60,6 +60,7 @@ public bool CanReplaceWithRValue(SemanticModel semanticModel, [NotNullWhen(true)
             var tokenParent = token.Parent;
             var sourceAncestor = tokenParent;
             // Skip the tuple expression if the identifier implies the name of the tuple field
+            // For example, we are evaluating `x` in `(x, y)`, but not in `(x: x, y: y)`
```

Comment:
i don't understand how the below check actually corresponds to the comment.  both of those cases would be identifier-names in an argument in a tuple.